tair/tair.go

```diff
@@ -2,9 +2,12 @@ package tair
 
 import (
 	"context"
+
 	"github.com/go-redis/redis/v8"
 )
 
+var _ TairCmdable = (*TairClient)(nil)
```

Comment:
why we need this line?

Reply:
```
This is a relatively common Go idiom equivalent to Java's "implements", Rust's "impl ... for", etc.
It

asserts that types implement expected set of interfaces (declaration of intent), guarding against breakage caused by method signature changes
improves readability by making things explicit & acting as an additional form of documentation
helps with code navigation/generation (e.g. in GoLand Alt+Insert on (*Type)(nil) gives Implement Missing Methods option).
```
https://github.com/uber-go/guide/issues/25

Reply:
thanks, got it.